_posts/2025-09-05-musical-physics-poster.markdown

```diff
@@ -0,0 +1,91 @@
+
+**Get started:** emphasise the collisions in the physics system by creating particle fx at the point of contact. Show direction and intensity so the user can see the sound being created.
+
+<video width=200 height=200 controls autoplay loop style="border-radius: 15px" alt="A small shape bouncing around the inside of a larger shape. Each collision creates lighting and particle effects.">
+  <source src="/assets/ADC_Visuals_Clip.mp4">
+</video>
+
+### There's always MIDI
+
+What would an audio app be without MIDI? Luckily there are open source and commercial plugins available for adding MIDI to Unity.
+If like us, you decide to add MIDI to your physics, watch out! It's really addictive! It's also easy to cause notes to get stuck on in your favourite synth.
+
+Take the time to ensure your note messages are balanced, always send a note off for every note on. These often get lost when objects get destroyed or the app pauses.
+
+### Experimental interactions
+
+Unity makes experimenting with interaction methods very simple. Want to hook into the system's accelerometer to make the shapes comply with real world gravity? There's a nice API for that. Pinches, clicks, drags and everything else - there's battle tested methods to implement different interactions and connect them to practically any parameter of the application.
+
+For example, swiping a finger over the shape in Playtonik will alter its angular velocity, as if you were spinning a globe.
+
+### Build for everywhere
+
+It's super simple to target multiple platforms, with support for most common platforms built into the engine. Any area that the built in tooling doesn't cover likely has a plugin tailor made by the vast Unity developer community.
+
+We've used plugins to handle [MIDI connectivity](https://assetstore.unity.com/packages/tools/audio/midi-plugin-for-mobile-and-desktop-198917) and [scales and notes](https://melanchall.github.io/drywetmidi/).
+
+### Going native
+
+Unity isn't a closed system, you can still work with native code when you need to. Handy for when there's platform specific configuration required or plugins that need system access.
+
+Need more audio control? There's a native audio system for that! Create audio code in C++ then bring back into Unity. Great for writing new audio effects.
+
+## The Drawbacks
+
+### Non-musical timing
+
+Unity is optimised towards visuals not audio. This has the consequence that the frame timing isn’t a great match for musical time - learn how we used threads to manage that for loop recording [here]().
+
+### Careful with audio quality
+
+There are a number of pitfalls when working with audio in Unity. Here are some top tips we've learn by falling into the traps!
+
+- If there are too many sound sources, things will be culled. Make sure it's the quietest sources using priorites.
+- Check those sample rates, on iOS Unity defaults to 24kHz - check this link for tips on how to run at 44.1kHz.
```

Comment:
Remove 'check this link'.

Perhaps just: "We found we had to manually force the sample rate via native code and in the Unity Project settings."